src/condor_utils/condor_crontab.cpp

```diff
@@ -884,6 +884,14 @@ CronTab::expandParameter( int attribute_idx, int min, int max )
 		// Sort! Makes life easier later on
 		//
 	this->sort( *list );	
+
+	// The topmost (DOW) can be empty, but if any other
+	// index generated an empty list, it indicates an error
+	// as an empty range passed in gets expanded to *
+
+	if ((attribute_idx != CRONTAB_DOW_IDX) && (0 == list->length())) {
+		return false;
```

Comment:
We should set CronTab::errorLog to a message here so that we can display what specifically went wrong in the hold reason for the job. Do note: After some testing there would need to be a line added into the schedd.cpp Scheduler::calculateCronTabSchedule() right before we delete the temporary CronTab object if the CronTab is not valid we should set `error = crontab->GetError();` Ask me if this is confusing.